volunteers/models.py

```diff
@@ -2,54 +2,99 @@
-    first_name = models.CharField(max_length=30, validators=[MinLengthValidator(1, "Please enter a valid name.")])
+    last_name = models.CharField(max_length=30, validators=[
+                                 MinLengthValidator(1, "Please enter a valid name.")])
+    first_name = models.CharField(max_length=30, validators=[
+                                  MinLengthValidator(1, "Please enter a valid name.")])
     middle_name = models.CharField(max_length=30)
-    gender = models.IntegerField
-    birth_date = models.DateTimeField('Date of birth')
-    physical_limitations = models.IntegerField(default=0)
-    physical_explanation = models.CharField(max_length=200)
-    street_address = models.CharField(max_length=50, validators=[MinLengthValidator(1, "Please enter a valid address.")])
-    state = models.CharField(max_length=2, validators=[MinLengthValidator(2, "Please enter a valid state abbreviation.")])
-    zip_code = models.CharField(max_length=5, validators=[MinLengthValidator(5, "Please enter a valid zipcode.")])
-    phone = models.CharField(max_length=12, validators=[MinLengthValidator(7, "Please enter a valid phone number (xxx-xxx-xxxx).")])
-    email = models.CharField(max_length=40, validators=[MinLengthValidator(4, "Please enter a valid email.")])
+
+    gender = models.CharField(
+        max_length=2,
+        choices=(
+            ("M", "Male"),
+            ("F", "Female"),
+            ("O", "Others")
+        )
+    )
+    birth_date = models.DateField('Date of birth')
+    # for birth dates, i would go with DateField
+    physical_limitations = models.CharField(
+        max_length=5,
+        choices=(
+                ("Y", "Yes"),
+                ("N", "No")
+        )
+    )
+    physical_explanation = models.TextField(blank=True, null=True)
+    street_address = models.CharField(max_length=50, validators=[
+                                      MinLengthValidator(1, "Please enter a valid address.")])
+    state = models.CharField(max_length=2, validators=[MinLengthValidator(
+        2, "Please enter a valid state abbreviation.")])
+    zip_code = models.CharField(max_length=5, validators=[
+                                MinLengthValidator(5, "Please enter a valid zipcode.")])
+    phone = models.CharField(max_length=12, validators=[MinLengthValidator(
+        7, "Please enter a valid phone number (xxx-xxx-xxxx).")])
+    email = models.EmailField(max_length=40, validators=[
+        MinLengthValidator(4, "Please enter a valid email.")])
     emergency_last_name = models.CharField(max_length=30)
     emergency_first_name = models.CharField(max_length=30)
     emergency_address = models.CharField(max_length=50)
-    emergency_phone = models.CharField(max_length=12, validators=[MinLengthValidator(7, "Please enter a valid phone number (xxx-xxx-xxxx).")])
+    emergency_phone = models.IntegerField(validators=[MinLengthValidator(
```

Comment:
Change to Charfield

Reply:
Can you explain why we should change this to charfield intead (interger, number, etc)?

Reply:
See comment above.